src/glue/resources/dataset_crawler_assignments.yaml

```diff
@@ -0,0 +1,10 @@
+standard:
+  - answers_v1
```

Comment:
@philerooski Please re-review -- crawler assignments now has list of versioned tables